### ISSUE TITLE
CSV import does not work with semicolon as separator

### ISSUE THREAD

**Issue by MagnusBieneck** [andrejev/Scalable.OR#2]
I was trying to import a CSV file that separates each column by semicola. The CSV import did not recognise the delimiter, so it imported the entire row as one column. In scalableor/core.py, one can observe that only the comma is recognised as delimiter (l.102):

```python
if args.add_import_command:
    or_program.insert(0, {"op": "scalableor/import", "separator": ",", "path": i_path})
```

I would suggest to add an optional parameter --csv-sep to change the default delimiter.

**Comment by MagnusBieneck** [comment]
I have just noticed that it is the same for CSV export. Hence, I will make `--csv-sep` apply for export as well.

**Comment by MagnusBieneck** [comment]
Issue solved #4
